source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,100 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can select and convert one or more queries, stored procedures, 
+triggers, and views in bulk by enabling the multi-select toggle. 
+Bulk conversion creates a job that queues and executes each query 
+conversion individually from a single request.
+
+Copy and paste your queries into the query converter individually and 
+then bulk convert a selection to update them to work with MongoDB and
+your migrated schema.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+- .. include:: /includes/fact-view-query-conversion-history.rst
```

Comment:
There is an extra status during bulk conversions, so you may need a different version of this include.

Reply:
Created a new include that has the pending status and modified the language to be a little more applicable. I'm referring to the pending icon as "Static circle" to differentiate from the spinner, but I'm curious if we have a more specific term for this UI element.